### PR TITLE
Add an "agent" to all of the masher messages.

### PR DESCRIPTION
**Pull request by ralphbean**
The idea here is that when a mash fails, we'll be able to easily tell _whose_
mash failed.

Then, we can change the IRC message from:
- bodhi masher failed to mash f24-updates-testing

to:
- bodhi masher failed on kevin's mash of f24-updates-testing

The benefit being that the releng team member on `.push_duty` will get a ping
in their IRC client when things fail so they can have a look.

---

Test suite is all passing here.

### CODE REVIEW

File: bodhi/tests/test_masher.py [hunk 1]
```diff
@@ -1009,7 +1028,7 @@ def test_push_timestamps(self, publish, *args):
         publish.assert_called_with(
             topic="mashtask.complete",
             force=True,
-            msg=dict(success=True, repo='f17-updates-testing'))
+            msg=dict(success=True, repo='f17-updates-testing', agent='lmacken'))
```
Comment (pypingou) [inline bodhi/tests/test_masher.py:1031]: This kinda surprises me a little since the MasherThread above seems to be created by `ralph`. Is this expected?

**Comment by lmacken** [comment]
Hmm, so right now people run the `bodhi-push` tool as the `apache` user in order to access the fedmsg certs. I think this would make the agent always be `apache`?

**Comment by ralphbean** [comment]
Looks like `bodhi-push` is able to figure out the name of the sudo-ing user:  https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.bodhi.masher.start  The agent looks correct in all those recent messages.

File: bodhi/tests/test_masher.py (hunk 1 above)
Comment (ralphbean) [inline bodhi/tests/test_masher.py:1031]: I think that one (where the `MasherThread` is created with `'ralph'`) is in a different test.  This one uses `self.masher` which is set to be `'lmacken'`.

**Comment by lmacken** [comment]
Sweet, sounds good then :)

**Comment by ralphbean** [comment]
woot!  Thanks @lmacken :)

**Pull request merged**